astropy/modeling/functional_models.py

```diff
@@ -854,6 +854,22 @@ def fit_deriv(x, amplitude, x_0, fwhm):
         d_fwhm = 2 * amplitude * d_amplitude / fwhm * (1 - d_amplitude)
         return [d_amplitude, d_x_0, d_fwhm]
 
+    def bounding_box(self, factor=25):
+        """Tuple defining the default ``bounding_box`` limits,
+        ``(x_low, x_high)``.
+
+        Parameters
+        ----------
+        factor : float
+            The multiple of FWHM used to define the limits.
+            Similar to `Gaussian1D`.
```

Comment:
Perhaps it's worth putting a sentence in explaining how the factor is chosen, e.g. just what you said here.